src/media_type.rs

```diff
@@ -172,6 +172,24 @@ impl fmt::Display for MediaType {
   }
 }
 
+impl From<MediaType> for deno_ast::MediaType {
```

Comment:
Should we have both of these or only use the one in `deno_ast`? Or perhaps `deno_ast` shouldn't even have a `MediaType`? Or maybe just doing this is ok?

Reply:
I think we refactor it to the most common location, so it should go in `deno_ast` IMO and no where else.  It is a bit of an argument if all of `deno_ast` would fit in `deno_graph`, but I don't mind the abstraction, and I think `MediaType` should really go into `deno_ast`, it is our "concept" of what makes a module and how we parse and handle it.

Reply:
I would re-export it from here though, so people can just use `deno_graph` without being aware of `deno_ast`.

Reply:
Done. Yeah I like the idea of just having one of these. I put it behind a feature flag.